### PR TITLE
HUBS-1561 GO SDK version update for DCT 6.0

### PR DESCRIPTION
**Pull request by kishlay-delphix**
This PR aims at updating the go-sdk to make it compatible with the latest version of DCT (6.0.0)
# Problem:

While updating the go sdk we encountered a number of issues:-

1. Duplicate entries were created for enums.
2. Codegen.sh used to override existing files, what this meant was even after a new generation we would have files which were already present and are not needed in the current version.

# Solution:

The above issues where solved in the following ways:-
1. Adding `-p enumClassPrefix=true` argument when using the jar to generate classes.
2. Cleanup of all existing go files and then generating all files.
# Testing

Tested get environments and VDBs.

### CODE REVIEW

File: generator-scripts/test.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 by Delphix. All rights reserved.
+#
+
+echo "$(dirname $0)"
```
Comment (eyalkaspi-delphix) [inline generator-scripts/test.sh:6]: ?

**Review by eyalkaspi-delphix — APPROVED** [review]

File: generator-scripts/test.sh (hunk 1 above)
Comment (kishlay-delphix) [reply generator-scripts/test.sh]: Deleted not needed file

**Review by sunnyx4 — APPROVED** [review]
1. can you put the JIRA issue in the subject? 
2. can you summarize the issues we had seen in the existing code-gen and the steps taken to resolve them here in the PR?

**Comment by kishlay-delphix** [comment]
@sunnyx4 Done
